backend/src/main/java/org/cryptomator/hub/api/GroupsResource.java

```diff
@@ -0,0 +1,38 @@
+package org.cryptomator.hub.api;
+
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.cryptomator.hub.entities.EffectiveGroupMembership;
+import org.cryptomator.hub.entities.Group;
+import org.cryptomator.hub.validation.ValidId;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+
+import java.util.List;
+
+@Path("/groups")
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
Overwritten by specific mediatypes in both methods

Reply:
Fixed in [b67cfdb](https://github.com/cryptomator/hub/pull/243/commits/b67cfdbb523156a8f6aced31e4997f3822e52a53)